testcase.py

```diff
@@ -1327,6 +1330,44 @@ def _get_full_current_state(self):
 
         return z
 
+    def store_results(self):
+        '''Stores results from scenario in working directory.
+        When ran with service, the result will be packed in the result tarball and
+        be retrieveable with the test_id
+
+
+        Returns
+        -------
+        None.
+
+        '''
+
+        name = "results"
+        # get results_json
+        results_json = {
+            "dateRun": str(datetime.now(tz=pytz.UTC)),
+            "boptestVersion": self.version,
+            "emulatorName": self.get_name()[2]['name'],
+            "controlStep": str(self.get_step()[2]),
+            "forecastParameters": {},  # for future use to store used parameters?
+            "measurementParameters": {},  # for future use to store used parameters?
```

Comment:
I have an initial implementation described here: https://github.com/ibpsa/project1-boptest/issues/514, but ran into an issue for truly accurate tracking as described at the end of that issue.

Reply:
Ok. I support the last proposed solution. But I guess this could be implemented first anyway?